commons/commons-serialization/src/test/kotlin/ac/uk/ebi/biostd/tsv/TsvToStringSerializerTest.kt

```diff
@@ -96,6 +97,30 @@ class TsvToStringSerializerTest {
         assertThat(testInstance.serialize(section)).isEqualToIgnoringNewLines(expectedTsv)
     }
 
+    @Test
+    fun `serialize subsection without accession`() {
```

Comment:
what about "serialize subsection without accession but the parent accession"  and not sure if we need test for ""serialize subsection with accession and parent accession"

Reply:
Done